packages/react/src/components/Accordion/AccordionItemDefer.test.jsx

```diff
@@ -44,4 +44,17 @@ describe('AccordionItemDefer', () => {
       screen.getByTestId('accordion-item-deferred').lastElementChild.childElementCount
     ).toEqual(1);
   });
+
+  it('triggers callback when heading is clicked', () => {
+    const headingClicked = jest.fn();
+    render(
+      <Accordion>
+        <AccordionItemDefer id="a" title="Title one" onHeadingClick={headingClicked}>
+          <p>This content</p>
+        </AccordionItemDefer>
+      </Accordion>
+    );
+    fireEvent.click(screen.getByRole('button', { name: 'Title one' }));
```

Comment:
nit: Would it be better to standardize on `userEvent` instead of `fireEvent` since it wraps in other parts of the interaction that fireEvent doesn't? focus before click for example?

Reply:
Yea, I normally use `userEvent` but this test file didn't so I went for consistency in the file.